app/src/main/java/dev/logickoder/jobfinder/home/HomeTopBar.kt

```diff
@@ -0,0 +1,78 @@
+package dev.logickoder.jobfinder.home
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Menu
+import androidx.compose.material3.CenterAlignedTopAppBar
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import dev.logickoder.jobfinder.app.theme.JobFinderTheme
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun HomeTopBar(modifier: Modifier = Modifier) {
+    CenterAlignedTopAppBar(
+        modifier = modifier,
```

Comment:
you missed padding the horizontal of the tab bar.

Reply:
Yeah, it didn't look so visually pleasing to me, so I removed it in the first place. But I've added it back though